packages/account/src/Sections/Security/Passwords/change-password-form.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik } from 'formik';
+import { PasswordInput, PasswordMeter } from '@deriv/components';
+import { useHistory, withRouter } from 'react-router-dom';
+import { routes, isMobile, useIsMounted, validPassword, validLength, getErrorMessages } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { WS } from 'Services/ws-methods';
+import { connect } from 'Stores/connect';
+import PasswordsFooter from './passwords-footer.jsx';
+
+const ChangePasswordForm = ({ email, is_trading_password, logout, onClickSendEmail }) => {
+    const [is_btn_loading, setIsBtnLoading] = React.useState(false);
+    const [is_submit_success, setIsSubmitSuccess] = React.useState(false);
+
+    const isMounted = useIsMounted();
+    const history = useHistory();
+
+    const handlePasswordChange = () => {
+        const params = {
+            action: 'redirect_to_login',
+            header: 'password_changed',
+        };
+        const search_params = new URLSearchParams(params).toString();
+
+        history.push({
+            pathname: routes.root,
+            search: `?${search_params}`,
+        });
+    };
+
+    const onSubmit = (values, { setSubmitting, setStatus, resetForm }) => {
+        setStatus({ msg: '' });
+        setIsBtnLoading(true);
+        if (is_trading_password) {
+            WS.tradingPlatformPasswordChange(values).then(data => {
+                if (isMounted()) {
+                    setIsBtnLoading(false);
+                    if (data.error) {
+                        setStatus({ msg: data.error.message });
+                        setSubmitting(false);
+                    } else {
+                        setIsSubmitSuccess(true);
+                        resetForm({ new_password: '' });
+                        setTimeout(() => {
+                            if (isMounted()) {
+                                setIsSubmitSuccess(false);
+                                setSubmitting(false);
+                            }
+                        }, 3000);
+                    }
+                }
+            });
+        } else {
+            WS.authorized.storage.changePassword(values).then(data => {
```

Comment:
Same idea here, since the state is being updated asynchronously, make sure the component is mounted.